src/chains.ts

```diff
@@ -1195,6 +1195,44 @@ export const OASIS_TESTNET: Chain = {
   blockExplorerUrls: ['https://testnet.explorer.emerald.oasis.dev'],
 }
 
+export const GODWOKEN_MAINNET: Chain = {
+  id: 'godwoken_mainnet',
+  chain_id: 71394,
+  name: 'Godwoken Testnet',
+  symbol: 'CKB',
+  mainnet: true,
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/godwoken.png',
+  pangolin_is_live: false,
+  tracked_by_debank: false,
+  supported_by_gelato: false,
+  rpc_uri: 'https://mainnet.godwoken.io/rpc',
+  nativeCurrency:{
+    name: 'Nervos Network',
+    symbol: 'CKB',
+    decimals: 18,
+  },
+  blockExplorerUrls: ['https://layerview.io'],
+}
+
+export const GODWOKEN_TESTNET: Chain = {
+  id: 'godwoken_mainnet',
+  chain_id: 71393,
+  name: 'Godwoken Testnet',
+  symbol: 'CKB',
+  mainnet: false,
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/godwoken.png',
+  pangolin_is_live: false,
+  tracked_by_debank: false,
+  supported_by_gelato: false,
+  rpc_uri: 'https://godwoken-testnet-web3-rpc.ckbapp.dev',
+  nativeCurrency:{
+    name: 'Nervos Network',
+    symbol: 'CKB',
+    decimals: 18,
+  },
+  blockExplorerUrls: ['https://aggron.layerview.io'],
```

Comment:
Should be https://aggron.gwscan.com/en-US

Reply:
https://v1.aggron.gwscan.com/ it seems to be this link (https://startwithnervos.com/godwoken)